src/components/pdfHelper/pdf-generate-form.js

```diff
@@ -155,11 +155,6 @@ export async function generateFormPDF(
     lineHeight: 16,
     maxWidth: 480,
   });
-  moveDownAndPopulateData(
-    "healthIndicatorQuestionnaire.contactForm.dateOnWhichDataWasCollected",
-    countrySummary.collectedDate
-  );
-  page.moveDown(20);
```

Comment:
Do we not need to add Updated date here in the PDF @preetighongal @laharigundu2001 @Kaveri-kmr ?

Reply:
Since we are not displaying updated date in the questionnaire I am not sure whether it is needed in PDF